apps/sms/views/shared/js/navigation.js

```diff
@@ -72,11 +71,11 @@ var Navigation = {
     this.transitionPromise = null;
 
     currentPanel = null;
+  },
 
-    Startup.on('post-initialize', function() {
```

Comment:
it's temporary code anyway, I don't think we'll need this isReady anymore once we have splitted panels